packages/haul-ram-bundle-webpack-plugin/src/__tests__/integration.test.ts

```diff
@@ -389,6 +389,6 @@ describe('File RAM bundle should build and successfully evaluate bundle', () =>
     );
 
     expect(mappedFrame1).toMatch('__fixtures__/nestedCrashFn.esm.js:4:9');
-    expect(mappedFrame2).toMatch('__fixtures__/crashFn.async.js:5:5');
+    expect(mappedFrame2).toMatch('__fixtures__/crashFn.async.js:5:16');
```

Comment:
is this expected?

Reply:
I think so, though I'm not 100% sure, `5` was pointing to
```
crashNested()
^
```
and `16`:
```
crashNested()
           ^
```